defaults.go

```diff
@@ -52,6 +53,23 @@ var RandomIdentity = func(cfg *Config) error {
 	return cfg.Apply(Identity(priv))
 }
 
+// DefaultListenAddrs configures libp2p to use default listen address
+var DefaultListenAddrs = func(cfg *Config) error {
+	defaultIP4ListenAddr, err := multiaddr.NewMultiaddr("/ip4/0.0.0.0/tcp/0")
+	if err != nil {
+		return err
+	}
+
+	defaultIP6ListenAddr, err := multiaddr.NewMultiaddr("/ip6/0.0.0.0/tcp/0")
```

Comment:
This should be `/ip6/::/tcp/0` (sorry for not noticing earlier). You'll also have to fix the test. `0.0.0.0` will use the IP4 over IP6 ALL address.

Reply:
aah, my bad.